### ISSUE TITLE
How to combine the project in my EVSE controller?...

### ISSUE THREAD

**Issue by foluswen** [eclipse/risev2g#4]
Hello Marc:

```
    Nice to see the project which implement V2G protocol.

    I can run secc & evcc on two computer, but how to apply the protocol on my EVSE controller?

   My EVSE controller can control hardware, but I don't know how to get communication message to change EVSE hardware action.

   Do  you have detail guide documents or sample code?
```

Best regards

Folus Wen

**Comment by MarcMueltin** [comment]
Hello Folus,

I did not have the opportunity yet to apply my RISE V2G code on a real EVSE hardware. In the "evseController" package (https://github.com/eclipse/risev2g/tree/master/RISE-V2G-SECC/src/main/java/org/eclipse/risev2g/secc/evseController) I programmed some generic interfaces to e.g. close or open a contactor and get metering information. As it is hardware dependant on how exactly the respective EVSE hardware gets controlled, it is up to you to code the concrete implementation.
In a previous prototype implementation of RISE V2G which I used back at the university, I did exactly that: programmed a class that implements that interface for a certain EVSE.

Did that information help you at least a bit?
